code/game/objects/effects/wanted_poster.dm

```diff
@@ -0,0 +1,56 @@
+/obj/item/weapon/poster/legit/wanted
+	var/poster_desc
+	icon_state = "rolled_poster"
+
+/obj/item/weapon/poster/legit/wanted/New(turf/loc, icon/person_icon, wanted_name, description)
+	..(loc)
+	name = "wanted poster ([wanted_name])"
+	desc = "A wanted poster for [wanted_name]."
+	poster_desc = description
+	qdel(resulting_poster)
+	if(person_icon)
+		var/obj/item/weapon/photo/photo = new()
+		photo.photocreate(null, icon(person_icon, dir = SOUTH))//copy this to prevent the cropping from happening to all the other images. There's probably a better way to copy an icon, but this works, so whatever.
+		resulting_poster = new /obj/structure/sign/poster/wanted(person_icon, wanted_name, poster_desc)
+		if(!resulting_poster || qdeleted(resulting_poster))
+			qdel(src)
+
+/obj/structure/sign/poster/wanted
+	var/wanted_name
+
+/obj/structure/sign/poster/wanted/New(var/icon/person_icon, var/person_name, var/description)
+	if(!person_icon)
+		qdel(src)
+		return
+	name = "wanted poster ([person_name])"
+	wanted_name = person_name
+	desc = description
+
+	var/obj/item/weapon/photo/photo = new /obj/item/weapon/photo()
+	photo.photocreate(null, icon(person_icon, dir = SOUTH))//copy the image so we don't mess with the one in the record.
```

Comment:
why not person_icon = icon(person_icon, dir = SOUTH)

why are you messing around with a photo obj? What benefit does it add?